### ISSUE TITLE
Firestore CLI setup - Error: Unexpected error encountered with rules.

### ISSUE THREAD

**Issue by newtriks** [firebase/firebase-tools#858]
### Version info

4.0.2

### Node version info

8.11.1

### Platform Information

OS X

### Steps to reproduce

1. `firebase init`
2. Select `firestore`
3. `? What file should be used for Firestore Rules? (firestore.rules)` - choose default option

### Expected behavior

Displays no error, creates the file and proceeds to the next step.

### Actual behavior

`Error: Unexpected error encountered with rules.`

```
 Sat Aug 04 2018 20:17:41 GMT+0100 (BST)
[debug] [2018-08-04T19:17:42.650Z] <<< HTTP RESPONSE 200
[info] 
[info] Firestore Security Rules allow you to define how and when to allow
[info] requests. You can keep these rules in your project directory
[info] and publish them with firebase deploy.
[info] 
[debug] [2018-08-04T19:17:47.877Z] >>> HTTP REQUEST GET https://firebaserules.googleapis.com/v1/projects/[my project name]/releases  
  
 Sat Aug 04 2018 20:17:47 GMT+0100 (BST)
[debug] [2018-08-04T19:17:48.713Z] <<< HTTP RESPONSE 200
[debug] [2018-08-04T19:17:48.713Z] [rules] error: 200
[error] 
[error] Error: Unexpected error encountered with rules.
```

### Notes

1. This works fine for other CLI features e.g. Functions
2. This works fine for node version 6.14.3
3. This appears to work fine for older projects

**Comment by panntomime** [comment]
I'm having the same issue
### Version info
4.0.2
### Node version info
v8.11.3
### Platform Information
Ubuntu 18.04 LTS

**Comment by ItangSanjana** [comment]
Same issue here but everything was perfect before Fri Aug 03 2018 21:52:11 GMT+0700 (WIB).

### Version info
Firebase cli 4.0.2
Node v8.11.3
NPM 6.3.0
OS Ubuntu 18.04 LTS

**Comment by jkusnier** [comment]
Same issue.

**Version info**

Firebase cli 4.0.2
Node v10.6.0
NPM 6.1.0
OS Windows 10 Version 1803 Build 17134.191

**Comment by mbleigh** [comment]
Thanks for the report, this should be fixed in `4.0.3`, just released.
